phpBB/styles/prosilver/template/forumlist_body.html

```diff
@@ -59,7 +66,11 @@
 							<!-- BEGIN subforum -->
 								<!-- EVENT forumlist_body_subforum_link_prepend -->
 								<a href="{forumrow.subforum.U_SUBFORUM}" class="subforum<!-- IF forumrow.subforum.S_UNREAD --> unread<!-- ELSE --> read<!-- ENDIF -->" title="<!-- IF forumrow.subforum.S_UNREAD -->{L_UNREAD_POSTS}<!-- ELSE -->{L_NO_UNREAD_POSTS}<!-- ENDIF -->">
-									<i class="icon <!-- IF forumrow.subforum.IS_LINK -->fa-external-link<!-- ELSE -->fa-file-o<!-- ENDIF --> fa-fw <!-- IF forumrow.subforum.S_UNREAD --> icon-red<!-- ELSE --> icon-blue<!-- ENDIF --> icon-md" aria-hidden="true"></i>{forumrow.subforum.SUBFORUM_NAME}</a><!-- IF not forumrow.subforum.S_LAST_ROW -->{L_COMMA_SEPARATOR}<!-- ENDIF -->
+									{{ Icon('iconify', {
+										'fa:external-link' : forumrow.subforum.IS_LINK ,
```

Comment:
There's a excessive space between `IS_LINK` and `,`.
```suggestion
										'fa:external-link' : forumrow.subforum.IS_LINK,
```